apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -216,28 +219,71 @@ public partial class SampleRowKeysStream
             }
         }
     }
-    
+
     public partial class BigtableServiceApiClientImpl
     {
         private BigtableServiceApiSettings _defaultSettings;
 
         /// <inheritdoc/>
-        public override BigtableServiceApiSettings DefaultSettings => _defaultSettings;
+        public override string AppProfileId => _defaultSettings.AppProfileId;
+
+        internal override BigtableServiceApiSettings DefaultSettings => _defaultSettings;
 
         partial void OnConstruction(
             Bigtable.BigtableClient grpcClient,
             BigtableServiceApiSettings effectiveSettings,
             ClientHelper clientHelper) =>
             _defaultSettings = effectiveSettings.Clone();
 
-        partial void Modify_MutateRowRequest(ref MutateRowRequest request, ref CallSettings settings) =>
+        // Note: the callbacks below intentionally take the specified request as the 1st argument so callers don't need
+        // to create closure objects at callsites.
+        private void TryApplyAppProfileId<TRequest>(
+            TRequest request,
+            Func<TRequest, string> appProfileIdGetter,
+            Action<TRequest, string> appProfileIdSetter)
+        {
+            if (AppProfileId != null && appProfileIdGetter(request).Length == 0)
+            {
+                appProfileIdSetter(request, AppProfileId);
+            }
+        }
+
+        partial void Modify_CheckAndMutateRowRequest(ref CheckAndMutateRowRequest request, ref CallSettings settings)
```

Comment:
Possibly use expression-bodied members for these where possible? https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Firestore.V1Beta1/Google.Cloud.Firestore.V1Beta1/FirestoreClientPartial.cs is very compact, for example? Definitely not a big deal though :)

Reply:
Right, meant to do this and forgot.